### ISSUE TITLE
Where is the `vscode-extension-common` in package.json dependencies?

### ISSUE THREAD

**Issue by vikyd** [dakaraphi/vscode-extension-fold#3]
I want to build vscode-extension-fold from source, but missing `vscode-extension-common`.

**Comment by d-akara** [comment]
@vikyd it is just a relative file reference in package.json.

So you will want to clone this repository https://github.com/dakaraphi/vscode-extension-common

**Comment by vikyd** [comment]
Why not publish [vscode-extension-common](https://github.com/dakaraphi/vscode-extension-common) to npm?

**Comment by d-akara** [comment]
I couldn't figure out a workflow that allowed for quick iterations.  I often make a lot of small quick changes between multiple plugins sharing this library and publishing to npm for each change would be a pain.

So using `npm link` should be a solution.  However, I ran into issues there as well.  Seems almost nobody is working with shared libraries pulling in vscode dependencies as it doesn't work.
These are some of the problems I ran into.
https://github.com/Microsoft/TypeScript/issues/6496
https://github.com/Microsoft/vscode/issues/25312
https://github.com/Microsoft/vscode-vsce/issues/207
